src/components/alphaPicker/alphaPicker.js

```diff
@@ -95,227 +106,228 @@ define(['focusManager', 'layoutManager', 'dom', 'css!./style.css', 'paper-icon-b
+                        element.addEventListener('click', onAlphaPickerInKeyboardModeClick);
+                    }
+
+                    if (options.valueChangeEvent !== 'click') {
+                        element.addEventListener('focus', onAlphaPickerFocusIn, true);
+                    } else {
+                        element.addEventListener('click', onAlphaPickerClick.bind(this));
+                    }
 
-                if (options.valueChangeEvent !== 'click') {
-                    element.addEventListener('focus', onAlphaPickerFocusIn, true);
                 } else {
-                    element.addEventListener('click', onAlphaPickerClick.bind(this));
-                }
 
-            } else {
+                    if (itemsContainer) {
+                        itemsContainer.removeEventListener('focus', onItemsFocusIn, true);
+                    }
 
-                if (itemsContainer) {
-                    itemsContainer.removeEventListener('focus', onItemsFocusIn, true);
+                    element.removeEventListener('click', onAlphaPickerInKeyboardModeClick);
+                    element.removeEventListener('focus', onAlphaPickerFocusIn, true);
+                    element.removeEventListener('click', onAlphaPickerClick.bind(this));
                 }
+            };
 
-                element.removeEventListener('click', onAlphaPickerInKeyboardModeClick);
-                element.removeEventListener('focus', onAlphaPickerFocusIn, true);
-                element.removeEventListener('click', onAlphaPickerClick.bind(this));
-            }
-        };
+            render(element, options);
 
-        render(element, options);
+            this.enabled(true);
+            this.visible(true);
+        }
 
-        this.enabled(true);
-        this.visible(true);
-    }
+        value(value, applyValue) {
 
-    AlphaPicker.prototype.value = function (value, applyValue) {
+            const element = this.options.element;
+            let btn;
+            let selected;
 
-        var element = this.options.element;
-        var btn;
-        var selected;
+            if (value !== undefined) {
+                if (value != null) {
 
-        if (value !== undefined) {
-            if (value != null) {
+                    value = value.toUpperCase();
+                    this._currentValue = value;
 
-                value = value.toUpperCase();
-                this._currentValue = value;
+                    if (this.options.mode !== 'keyboard') {
+                        selected = element.querySelector(`.${selectedButtonClass}`);
 
-                if (this.options.mode !== 'keyboard') {
-                    selected = element.querySelector('.' + selectedButtonClass);
+                        try {
+                            btn = element.querySelector(`.alphaPickerButton[data-value='${value}']`);
+                        } catch (err) {
+                            console.error(`error in querySelector: ${err}`);
```

Comment:
I prefer splitting the error with a comma for logs.

Reply:
Mentioning "error in" in an error level log is also a bit redundant :p